lib/ansible/plugins/test/compat_jinja_2_10.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017, the Jinja Team
+# BSD 3-Clause "New" or "Revised" License (see https://opensource.org/licenses/BSD-3-Clause)
```

Comment:
I'm not sure if this is compatible with the GPLv3 we use in Ansible.

Reply:
> I'm not sure if this is compatible with the GPLv3 we use in Ansible.

Jijna itself is licenced with BSD 3-Clause (new BSD?) [0].

\[0]: https://github.com/pallets/jinja/blob/master/LICENSE.rst

Reply:
> I'm not sure if this is compatible with the GPLv3 we use in Ansible

Looked it up: https://en.m.wikipedia.org/wiki/License_compatibility#GPL_compatibility

> Many of the most common free-software licenses, especially the permissive licenses, such as the original MIT/X license, BSD licenses (in the three-clause and two-clause forms, though not the original four-clause form), MPL 2.0, and LGPL, are GPL-compatible.

Hence, I can apply the GPLv3 label of it helps get it merged.